data/json/monsters/feral_humans.json

```diff
@@ -58,6 +58,65 @@
       "DROPS_AMMO"
     ]
   },
+  {
+    "id": "mon_feral_sapien_spear",
+    "type": "MONSTER",
+    "name": { "str": "feral sapien" },
+    "description": "In this feral's eyes is more intelligence than in those of most of their kind.  They appear to be holding a spear.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 2,
+    "dodge": 3,
+    "armor_bash": 9,
+    "armor_cut": 7,
+    "armor_bullet": 12,
+    "special_attacks": [
+      {
+        "id": "stretch_attack",
+        "cooldown": 5,
+        "range": 2,
+        "hit_dmg_u": "The %1$s thrusts a spear at your %2$s, slashing it!",
+        "hit_dmg_npc": "The %1$s thrusts a spear at <npcname>'s %2$s, slashing it!",
+        "no_dmg_msg_u": "The %1$s thrusts a spear at your %2$s, but glances off your armor!",
+        "no_dmg_msg_npc": "The %1$s thrusts a spear at <npcname>'s %2$s, but glances off armor!",
+        "miss_msg_u": "The %s thrusts a spear at you, but you evade it!",
+        "miss_msg_npc": "The %s thrusts a spear at <npcname>, but they evade it!"
+      }
+    ],
```

Comment:
Without adding a PARROT special attack, they're using the lines? I am confused.

Reply:
> Without adding a PARROT special attack, they're using the lines? I am confused.

Thought I forgot something.